config/environments/follower.rb

```diff
@@ -10,4 +10,5 @@
   config.log_level = ENV['CONJUR_LOG_LEVEL'] || :info
   config.middleware.use Rack::RememberUuid
   config.audit_socket = '/run/conjur/audit.socket'
+  config.audit_database = 'postgres://:5433/audit'
```

Comment:
don't understand the logic, please explain

Reply:
This setting take precedence, we are overriding the value of config.audit_database.
The follower should now write the audit directly instead of syslog-ng